site/documentation/content/user-guide/mqtt-adapter.md

```diff
@@ -42,6 +42,9 @@ After verifying the credentials, the number of existing connections is checked a
 
 Before accepting any telemetry or event or command messages, the MQTT adapter verifies that the configured [message limit] ({{< relref "/concepts/resource-limits.md" >}}) is not exceeded. The incoming message is discarded if the limit is exceeded. 
 
+## Connection Event
+The MQTT Adapter can send a [Connection Event]({{< relref "/api/event#connection-event" >}}) at once the connection with a device has been successfully established or ended. Note that this requires the [`HONO_CONNECTION_EVENTS_PRODUCER`]({{< relref "/admin-guide/mqtt-adapter-config#service-configuration" >}}) configuration property to be explicitly set to `events`.
```

Comment:
... ~~at~~ once the connection ...

Reply:
@sophokles73 I have updated and pushed the changes. Pls have a look